docs/build/isc/v1.0.0-rc.6/docs/getting-started/quick-start.mdx

```diff
@@ -56,6 +56,12 @@ If you want to fund your EVM testnet account, please refer to our [How To Get Fu
 
 You can now use your testnet tokens and simulated bridged tokens to deploy and interact with smart contracts on the testnets. Utilize popular development tools and frameworks, such as [Hardhat](https://hardhat.org/), or [Remix](https://remix.ethereum.org/), to build, test, and deploy your smart contracts.
 
+:::tip Deploy a Smart Contract
```

Comment:
Please replace this with an imported admonition from https://github.com/iotaledger/iota-wiki/blob/docs/update-quick-start-guide/docs/build/isc/v1.0.0-rc.6/docs/_admonitions/_deploy_a_smart_contract.md

Reply:
Ok thanks, on it...

Reply:
@lucas-tortora, I have made the change.